src/test/csi-qingcloud/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+#  Copyright (C) 2018 Yunify, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this work except in compliance with the License.
+#  You may obtain a copy of the License in the LICENSE file, or at:
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+apiVersion: v2
+
+appVersion: 1.2.0
+name: csi-qingcloud
+description: A Helm chart for Qingcloud CSI Driver
+version: 1.2.0-rc1
```

Comment:
Can we use a formal version without `-rc1`? Helm v3 ignores pre-releases by default.

Reply:
ok